URLtoPebblePng.py

```diff
@@ -0,0 +1,49 @@
+import png
+import feedparser
+import itertools
+import Image
+import urllib2
+import ctypes
+from HTMLParser import HTMLParser
+from PIL import ImageEnhance
+import argparse
+
+#Get image from URL and generate pebble compliant png
+def get_pebble_png(input_url):
+    img = urllib2.urlopen(input_url)
+    localFile = open('TempImage.jpg', 'wb')
```

Comment:
The tempfile module didn't work for an image file. It couldn't find the size parameters for the temporary file. But I updated the code to delete the TempImage after it's completed conversion